README.md

```diff
@@ -39,10 +39,11 @@ Open main.py which contains a simple starter example.
 The ```ContractUtility``` class is used to compile and deploy the contracts, 
 based on the network name (sapphire, sapphire-testnet, sapphire-localnet).
 The private key used to deploy the contract is fetched from the PRIVATE_KEY 
-environment variable.
+environment variable. If you intend to use async methods and websocket
+subscription set the ```use_async_w3=True```.
 
 ```python
-contract_utility = ContractUtility("sapphire-localnet")
+contract_utility = ContractUtility("sapphire-localnet", use_async_w3=False)
```

Comment:
Is there a reason for keeping the non-async version of the function? I would simply make the `deploy_contract()` async without too much hassle.

Reply:
The deploy_contract() is now async. Some updates were also needed to the [main.py/ContractUtility/utils.py ](https://github.com/oasisprotocol/demo-starter-py/pull/6/commits/0aafe7ffb61a7b2600d84418f39b8e34477c0496) and tests, I commented out the sync parts for review. The CLI now uses async deploy_contract().

Reply:
Why `use_async_w3=False` ? Can't we just leave default as a preferred flow? If async is disabled, then the Async method below under Deploying the contract also won't work?

Reply:
Removed use_async, the demo now uses async by default.